site/docs/tutorials/spring-boot.md

```diff
@@ -117,33 +117,54 @@ hazelcast-jet:
     queue-size: 2048
 ```
 
+To configure the underlying `HazelcastInstance` we'll define a
+configuration file named `hazelcast.yaml` at the root directory.
+
+```yaml
+hazelcast:
+  cluster-name: tutorial-jet-starter
+```
+
 When you stop and re-run the main class you should now see that the
-configuration file we've just created is used to start the member:
+configuration files we've just created is used to start the member:
 
 ```text
 ...
 c.h.i.config.AbstractConfigLocator       : Loading 'hazelcast-jet.yaml' from the working directory.
 ...
+c.h.i.config.AbstractConfigLocator       : Loading 'hazelcast.yaml' from the working directory.
+...
 ```
 
 ### Using Properties File
 
-If your configuration file is not at the root directory or you want to
+If your configuration files are not at the root directory or you want to
 use a different name then you can create an `application.properties`
-file and set the `hazelcast.jet.config` like below:
+file and set the `hazelcast.jet.server.config` and `hazelcast.jet.imdg.config`
+like below:
 
 ```properties
-hazelcast.jet.config=file:config/hazelcast-jet-tutorial.yaml
+hazelcast.jet.server.config=file:config/hazelcast-jet-tutorial.yaml
+hazelcast.jet.imdg.config=file:config/hazelcast-tutorial.yaml
 ```
 
+Since Spring Boot converts these config properties to resource URLs,
+you need to use `file:` prefix for files at the working directory and
+`classpath:` for files on the classpath.
+
 ### Using System Properties
 
-You can also set configuration file using system property:
+You can also set configuration files using system property:
 
 ```java
-System.setProperty("hazelcast.jet.config", "file:config/hazelcast-jet-tutorial.yaml");
+System.setProperty("hazelcast.jet.config", "config/hazelcast-jet-tutorial.yaml");
+System.setProperty("hazelcast.config", "config/hazelcast-jet-tutorial.yaml");
```

Comment:
It's just a detail but I think it's better to use different name for values of `hazelcast.jet.config` and `hazelcast.config` to have it more clear that they are actually different files. Please use `config/hazelcast-tutorial.yaml` for `hazelcast.config`.